third_party/npm_@trezor_connect-web/README.chromium

```diff
@@ -0,0 +1,3 @@
+Name: trezor_connect-web
+URL: https://github.com/trezor/trezor-suite
+License: T-RSL
```

Comment:
When dealing with custom licenses, the [SPDX way](https://spdx.github.io/spdx-spec/v2.3/other-licensing-information-detected/) to express that is `LicenseRef-T-RSL`, like we do in https://github.com/brave/brave-core/blob/master/third_party/android_deps/libs/com_android_billingclient/README.chromium.
```suggestion
License: LicenseRef-T-RSL
```

Reply:
Sure. I just copied what we have in https://github.com/brave/brave-core/blob/master/third_party/npm_%40trezor_connect/README.chromium (added in https://github.com/brave/brave-core/commit/b2d6b3cf649aacff836ba6376abe76a34d1a4f9c), so I can modify that one as well.